packages/backend/src/controllers/eig/get-pdf.js

```diff
@@ -0,0 +1,47 @@
+const Eig = require("../../services/eig");
+const PdfEig = require("../../services/pdf/eig/generate");
+const logger = require("../../utils/logger");
+const Region = require("../../services/geo/Region");
+const AppError = require("../../utils/error");
+
+const log = logger(module.filename);
+
+module.exports = async function getPdf(req, res, next) {
+  const { id: eigId } = req.params;
+  log.i("IN", { eigId }, req.body);
+  if (!eigId) {
+    log.w("missing parameter");
+    return next(
+      new AppError("Paramètre incorrect", {
+        statusCode: 400,
+      }),
+    );
+  }
+  try {
+    const eig = await Eig.getById({ eigId });
+    if (!eig) {
+      return next(
+        new AppError("Eig non trouvé", {
+          statusCode: 404,
+        }),
+      );
+    }
+
+    const serviceRegional = await Region.fetchOne(eig.agrementRegionObtention);
+    if (!serviceRegional) {
+      return next(
+        new AppError("Service régional de l'agrément non trouvé", {
+          statusCode: 404,
+        }),
+      );
+    }
+    const file = await PdfEig({
```

Comment:
```suggestion
    const file = await generatePdfEig({
```
pour moi une variable qui commence par une majuscule est une classe ou un enum. ici vue que c'est une méthode il vaux mieux que ça commence par une minuscule et que ça explicite l'action réalisé.
Dans le futur en TS, il sera intéressant de favorisé des export nommé pour que chaque import ne donne pas un nom différent à une meme fonction.